joomla/joomla-cms
# [5.0] Include article filter

Alternative Pull Request for Issue #41745 .

### Summary of Changes
Add the ability to restict the mod_articles_category module to display the included articles


### Testing Instructions
Publish the module and check that is displaying a lot of articles
Take the id of one of those articles and enter it as the ID to Exclude
Check that the module is now NOT displaying that article
Remove the ID from the exclude filter and put it in the include filter instead
Check that the module is now ONLY displaying that article

### Additional comment
This was made to show what should have been done in #41745 I am not commenting if its a useful change or not.
